completion/services.py

```diff
@@ -118,7 +118,7 @@ def vertical_is_complete(self, item):
             return None
 
         # this is temporary local logic and will be removed when the whole course tree is included in completion
```

Comment:
Gotta love "temporary" commits that are more than a few years old :stuck_out_tongue: